### PR TITLE
add folder jsBasics with files home work #3

### PR DESCRIPTION
**Pull request by ShostVladyslav**

### CODE REVIEW

File: jsBasics/task1.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const userName = "Semen"
+const age = 22
+const isStudent = true
+const phoneNumber = null
+const email =
```
Comment (sttaran) [inline jsBasics/task1.js]: when you use const you should assign value immediately

Comment (sttaran) [reply jsBasics/task1.js]: but you can use `let`

File: jsBasics/task3.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let ageSemen = 22
+let isAdult = ageSemen >= 18
```
Comment (sttaran) [inline jsBasics/task3.js]: use const

**Review by sttaran — CHANGES_REQUESTED** [review]

File: jsBasics/task1.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const userName = "Semen"
+const age = 22
+const isStudent = true
+const phoneNumber = null
+let email =
```
Comment (sttaran) [inline jsBasics/task1.js]: remove =

**Review by sttaran — CHANGES_REQUESTED** [review]

File: jsBasics/task1.js (hunk 3 above)
Comment (ShostVladyslav) [reply jsBasics/task1.js]: done

**Review by sttaran — APPROVED** [review]

**Pull request merged**